community/kernel/src/test/java/org/neo4j/kernel/api/index/IndexProviderCompatibilityTestSuite.java

```diff
@@ -50,6 +56,14 @@
 {
     protected abstract SchemaIndexProvider createIndexProvider( PageCache pageCache, FileSystemAbstraction fs, File graphDbDir );
 
+    public abstract Iterable<Value> getSupportedValues();
+
+    protected List<Value> commonValues = Arrays.asList( Values.of( "string1" ), Values.of( 42 ) );
```

Comment:
WIth these values and what we discussed @fickludd , I was refering to [FusionIndexTestHelp](https://github.com/neo4j/neo4j/blob/3.4/community/kernel/src/test/java/org/neo4j/kernel/impl/index/schema/fusion/FusionIndexTestHelp.java) and a potential co-usage of those lists somehow so that we have one place with utilities for all supported value types.

More of a nice-to-have really.

Reply:
Yupp. We made a card for cleaning around there.